CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Master
+* Remove Codecov token from .travis.yml and put it (after regeneration) in Travis repository settings
```

Comment:
I don't think it's required to put it in travis repository settings, as indicated in codecov project setting page

```
Note: Token not required for public repositories uploading from Travis, CircleCI or AppVeyor.
````

```suggestion
* Remove Codecov token from .travis.yml
```

Reply:
Hum so let's try it :) Thanks